src/test/fixtures/sample-config.json

```diff
@@ -603,6 +652,14 @@
         "remarksColor": "white",
         "title": "Documents Bundle",
         "remarks": "Some very important documents in here for some submission",
+        "misc": {
+          "miscKey1": "miscValue1",
+          "uiSchema": {
+            "remarks": {
+              "ui:widget": "textarea"
+            }
+          }
+        },
```

Comment:
![image](https://user-images.githubusercontent.com/7406870/97254236-e7c39a80-1848-11eb-91e2-89beb281ddf2.png)


Hmm what's the impact of this? From the forms perspective I don't see an additional textarea.

Reply:
This is to show that if the data has a uiSchema field it wont be affected is it?

Reply:
Yep - I thought that's what you meant by this: https://github.com/TradeTrust/document-creator-website/pull/62#discussion_r501427631

(correct me if I'm wrong)